src/main/java/com/epam/reportportal/jbehave/ReportPortalStoryReporter.java

```diff
@@ -280,12 +280,12 @@ protected StartTestItemRQ buildStartExampleRq(@Nonnull final Map<String, String>
 	 * Return parameter names used in bypassed step
 	 *
 	 * @param step a step name pattern
-	 * @return a list of parameter names from the step
+	 * @return a set of parameter names from the step
 	 */
 	@Nonnull
-	protected List<String> getUsedParameters(@Nonnull final String step) {
+	protected Set<String> getUsedParameters(@Nonnull final String step) {
```

Comment:
Unfortunately now I see an issue with using `Set` here. In an Example someone can specify same parameter values under different names. And it will lead to parameters miss on Report Portal. I understand that I collect these parameters into `Map` later, but now I would like to change it.

Reply:
@HardNorth  could you please elaborate? Per my understanding here the logic is parsing examples parameters keys from the step like this: '<parameter-key>'.

Reply:
@ikalinin1 Ahh, nevermind. I confused a case.